data/xml/2020.icon.xml

```diff
@@ -40,11 +1061,11 @@
       <author><first>Kranti</first><last>Ch</last></author>
       <author><first>Bharath</first><last>Challa</last></author>
       <author><first>Sonal</first><last>Kumari</last></author>
-      <author><first></first><last>Harshavardhana</last></author>
+      <author><first>Harshavardhana</first><last>.</last></author>
```

Comment:
This doesn’t look quite right.

Reply:
The PDF shows `Harshavardhana`
<img width="466" alt="Screen Shot 2021-07-12 at 9 34 58 PM" src="https://user-images.githubusercontent.com/27436920/125376546-3a060680-e359-11eb-91d6-84922bb061cb.png">

Reply:
Right but that should be in the <last> field

Reply:
The line before this change was correct, single names should be in `<last>` with an empty `<first>` element

Reply:
Got it, wasn't aware of that. Thank you for both!

Reply:
Sorry, I did this on mobile, and wrote `<last>`, but since I didn't use backticks, I see that Slack just treated it as an unknown tag and made it invisible!